### PR TITLE
v9 upgrade

### PR DESCRIPTION
**Pull request by FloppyDisck**
Added bumped archway versions which should now allow up to implement the new modules.

### CODE REVIEW

File: packages/archway-test-tube/libarchway/testenv/setup.go [hunk 1]
```diff
@@ -202,6 +202,7 @@ func InitChain(appInstance *app.ArchwayApp) (sdk.Context, secp256k1.PrivKey) {
 	requireNoErr(err)
 
 	concensusParams := simtestutil.DefaultConsensusParams
+	// TODO: max gas could be set to 300mil
```
Comment (pyncz) [inline packages/archway-test-tube/libarchway/testenv/setup.go:205]: So, why not..? 😅

**Review by pyncz — COMMENTED** [review]
Those `.doSomethingLegacy` methods scare me a bit, is there something deprecated under the hood?

**Comment by FloppyDisck** [comment]
> Those `.doSomethingLegacy` methods scare me a bit, is there something deprecated under the hood?

Its an annoying convention cosmos has, the new functions are literally  wrappers for the legacy function to provide less functionality.

```go
// NewContext returns a new sdk.Context with a empty header
func (app *BaseApp) NewContext(isCheckTx bool) sdk.Context {
	return app.NewContextLegacy(isCheckTx, cmtproto.Header{})
}
```

In the case of this example and `LegacyContext` I actually need to modify the `cmproto.Header`

File: packages/archway-test-tube/libarchway/testenv/setup.go (hunk 1 above)
Comment (FloppyDisck) [reply packages/archway-test-tube/libarchway/testenv/setup.go:205]: I wanted to leave it for another issue since its not the context of this PR

**Review by pyncz — APPROVED** [review]

**Pull request merged**